tensorboard/components/vz_chart_helpers/plottable-interactions.js

```diff
@@ -0,0 +1,144 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+var vz_chart_helpers;
+(function(vz_chart_helpers) {
+  // HACK: parentElement does not work for webcomponents.
```

Comment:
Can you include a link to https://github.com/palantir/plottable/issues/3350 somewhere?